classes/notification/NotificationManagerDelegate.php

```diff
@@ -56,7 +56,7 @@ public function updateNotification(PKPRequest $request, ?array $userIds, int $as
      *
      * @copydoc PKPNotificationOperationManager::createNotification()
      */
-    public function createNotification(PKPRequest $request, ?int $userId = null, ?int $notificationType = null, ?int $contextId = Application::SITE_CONTEXT_ID, ?int $assocType = null, ?int $assocId = null, int $level = Notification::NOTIFICATION_LEVEL_NORMAL, ?array $params = null): ?Notification
+    public function createNotification(?PKPRequest $request = null, ?int $userId = null, ?int $notificationType = null, ?int $contextId = Application::SITE_CONTEXT_ID, ?int $assocType = null, ?int $assocId = null, int $level = Notification::NOTIFICATION_LEVEL_NORMAL, ?array $params = null): ?Notification
```

Comment:
do we have any use of `PKPRequest $request` here as it seems this data not used ?

Reply:
You're right -- this parameter can be removed.